appcli/commands/service_cli.py

```diff
@@ -27,6 +32,20 @@
 # ------------------------------------------------------------------------------
 
 
+class ServiceAction(enum.Enum):
+    """
+    Enum representing avaliable actions to apply to an appcli service or group of services.
+    This enum is required to be passed into __action_runner().
```

Comment:
While the context of using this enum is limited (i.e. should only be used in this class), this type of documentation will become invalid very easily. You want to keep your doco comments accurate but not highly coupled to the calling code.